certbot/certbot/_internal/cli/__init__.py

```diff
@@ -171,11 +171,8 @@ def prepare_and_parse_args(plugins: plugins_disco.PluginsRegistry, args: List[st
     helpful.add(
         ["register", "automation"], "--register-unsafely-without-email", action="store_true",
         default=flag_default("register_unsafely_without_email"),
-        help="Specifying this flag enables registering an account with no "
-             "email address. This is strongly discouraged, because you will be "
-             "unable to receive notice about impending expiration or "
-             "revocation of your certificates or problems with your Certbot "
-             "installation that will lead to failure to renew.")
+        help="Specifying this flag skips prompting for an email address when running"
```

Comment:
We can probably just make this a hidden flag now, since this is the default behavior. We want to keep it so that old scripts don't break but I can't imagine anyone actually needing to know about it with this change.